blocks/fragment/fragment.js

```diff
@@ -0,0 +1,30 @@
+import {
+  decorateMain,
+  loadBlocks,
+} from '../../scripts/scripts.js';
+
+export default async function decorate(block) {
+  const ref = block.textContent.trim();
+  const path = new URL(ref, window.location.href).pathname.split('.')[0];
+  const resp = await fetch(`${path}.plain.html`);
+  if (resp.ok) {
+    const main = document.createElement('main');
+    main.innerHTML = await resp.text();
+    const img = main.querySelector('img');
```

Comment:
Why only first image is lazy loaded ?

Reply:
Because the rest is set to`lazy` anyway. But to be honest, at this point it's irrelevant anyway, so I'll remove it again.